### PR TITLE
Pass error frames out of handler

### PR DESCRIPTION
**Pull request by jcorbin**
Prep before #15, which will continue work in `TChannelConnection#onErrorFrame`, and not always construct an error object. 

r @Raynos @rf

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ TChannelConnection.prototype.setupHandler = function setupHandler() {
 
     self.handler.writeErrorEvent.on(onWriteError);
     self.handler.errorEvent.on(onHandlerError);
+    self.handler.errorFrameEvent.on(onErrorFrame);
     self.handler.callIncomingRequestEvent.on(onCallRequest);
     self.handler.callIncomingResponseEvent.on(onCallResponse);
     self.handler.pingIncomingResponseEvent.on(onPingResponse);
```
Comment (Raynos) [inline connection.js]: please `callIncomingErrorEvent` -> `callIncomingErrorFrameEvent`

**Comment by Raynos** [comment]
good looks good. please fix name of thingy :)

**Comment by jcorbin** [comment]
Renamed.

**Comment by Raynos** [comment]
ship

**Pull request merged**